src/gluonnlp/model/transformer.py

```diff
@@ -594,6 +624,10 @@ class TransformerEncoderCell(BaseTransformerEncoderCell):
         Prefix for name of `Block`s. (and name of weight if params is `None`).
     params : Parameter or None
         Container for weight sharing between cells. Created if `None`.
+    activation : str, default None
```

Comment:
Should the default value be relu instead? If someone uses TransformerEncoderCell API to contruct a cell directly with the default value, it should set act='relu'. 

Would you mind also adding a unit test for TransformerEncoderCell in https://github.com/dmlc/gluon-nlp/blob/master/tests/unittest/test_models.py#L543 with some dummy inputs, to check if the cell with default arguments can be constructed?

Reply:
The default behaviour would de decide upon whether he uses BERTPositionwiseFFN or PositionwiseFFN
`positionwise_ffn = BERTPositionwiseFFN if use_bert else PositionwiseFFN`
https://github.com/paperplanet/gluon-nlp/blob/894b60b2e513606a0fa3ff2c55d8756c4eb8f7d7/src/gluonnlp/model/transformer.py#L246

Reply:
Yes.
Added a simple test.